src/coffee/services/product-projections.coffee

```diff
@@ -250,7 +250,6 @@ class ProductProjectionService extends BaseService
   #   .search()
   search: ->
     @_currentEndpoint = '/product-projections/search'
-    @fetch()
```

Comment:
Actually it should work https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/services/base.coffee#L329
Let me have a look.